### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by jpol01**
## Add Contact Form and finish the mobile version.

>Objectives:
        - Add a contact form to my project and my partner project too.
        - Use the hml5 validations.
        - Understand how to parse Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -312,5 +313,51 @@ <h3 class="about-me-h3">About me</h3>
...
+              name="message"
+              required
+            ></textarea>
+          </li>
+          <li class="li-button">
+            <button class="about-me-button" type="submit">Get in touch</button>
+          </li>
+        </ul>
+      </form>
+    </footer>
```
Comment (mike2611) [inline index.html:361]: - [ ] Please make sure to follow the template as close as possible, I see that you are missing the bottom part of the template you choose. please let's fix this :+1: 
## Figma Template
image
## Your Project
image

File: index.html [hunk 2]
```diff
@@ -25,6 +25,7 @@
 
     <title>Porfolio</title>
```
Comment (mike2611) [inline index.html:26]: [OPTIONAL]

- [ ] Please fix this typo in your `title` :computer:

**Review by mike2611 — CHANGES_REQUESTED** [review]
Hi @jpol01 and @joshuaivie , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jpol01) [reply index.html:404]: Hi, I added the missing part. Thanks for your review. 
I honestly don't know how I skipped it
:smile_cat: 

I'm adding a screenshot of my project now:

to2o

File: index.html (hunk 2 above)
Comment (jpol01) [reply index.html]: Hi I fixed the typo,  thanks again for your review :smile_cat:

**Review by elmar8287 — APPROVED** [review]
Hi team , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**